ibis/tests/all/test_numeric.py

```diff
@@ -304,6 +305,7 @@ def test_binary_arithmetic_operations(backend, alltypes, df, op):
     )
 
 
+@pytest.mark.skip_backends((MSSQL,))
```

Comment:
any reason to use `skip_backends` here instead of `xfail_backends`?

Reply:
no reason at all. changed to xfail_backends